mne/minimum_norm/inverse.py

```diff
@@ -785,6 +785,10 @@ def apply_inverse(evoked, inverse_operator, lambda2=1. / 9.,
     #
     #   Pick the correct channels from the data
     #
+    # XXX _pick_channels_inverse_operator does a set-like operation, so
+    # this (and all other calls with _pick_channels_inverse_operator)
+    # will fail if users have reordered their channels
+    # (or if e.g., one recording has a different order from another)
```

Comment:
Instead of throwing an error, we could probably support the use case by using a picking function that respected the order of the input. Would that be better?